src/Microsoft.Extensions.Configuration/ConfigurationSection.cs

```diff
@@ -30,8 +38,14 @@ public ConfigurationSection(ConfigurationRoot root, string path)
             _path = path;
         }
 
+        /// <summary>
+        /// Gets the full path to this section within the <see cref="IConfiguration"/>.
```

Comment:
`Gets the full path to this section from the <see cref="IConfigurationRoot"/>. `